kernel/arch/amd64/src/asm.S

```diff
@@ -78,7 +78,7 @@ FUNCTION_END(memcpy_to_uspace)
 
 SYMBOL(memcpy_from_uspace_failover_address)
 SYMBOL(memcpy_to_uspace_failover_address)
-	xorl %eax, %eax         /* return 0, failure */
```

Comment:
Shouldn't really be necessary on amd64 as it will zero-extend the result to the full 64-bit register

Reply:
Can't hurt either, though.